Gemfile

```diff
@@ -44,6 +44,8 @@ gem 'sitemap_generator'
 # Makes sending HTML emails a little less painful by inlining stylesheets and rewriting relative URLs
 gem 'roadie-rails', '~> 1.0.4'
 
+gem 'friendly_id', '~> 5.1.0'
```

Comment:
It is a good idea to provide a comment what this gem is for and sometimes even provide a link to the repo on GitHub or documentation. See all the other gems above that.